common/reviews/api/rush-lib.api.ts

```diff
@@ -496,19 +517,30 @@ class VersionMismatchFinder {
 class VersionPolicy {
   constructor(versionPolicyJson: IVersionPolicyJson);
   // (undocumented)
+  public abstract bump(bumpType?: BumpType, identifier?: string): void;
```

Comment:
These are optional because these are only used to override what policy has defined. 

---
In reply to: [134069787](https://github.com/Microsoft/web-build-tools/pull/277#discussion_r134069787) [](ancestors = 134069787)